File: shanny-blog-java/blog-pojo/src/main/java/com/shanny/dto/CategoryDTO.java
package com.shanny.dto;

import com.shanny.enums.CategoryEnum;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

@Data
@Schema(description = "添加菜单时提交的数据")
public class CategoryDTO {
    private Long id;
    private String name;
    private String nameEn;
    private CategoryEnum.CategoryType type;
    private int sort;
}

File: shanny-blog-net/blog-pojo/Dtos/CategoryDTO.cs
using blog_common.Enums;

namespace blog_pojo.Dtos
{
    public class CategoryDTO
    {
        public long Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string NameEn { get; set; } = string.Empty;
        public CategoryType Type { get; set; }
        public int Sort { get; set; }
    }
}
